graphql/resolvers.go

```diff
@@ -744,6 +744,27 @@ func (r *taskResolver) PatchNumber(ctx context.Context, obj *restModel.APITask)
 	return &patch.PatchNumber, nil
 }
 
+func (r *taskResolver) BaseCommitDuration(ctx context.Context, at *restModel.APITask) (*restModel.APIDuration, error) {
+	baseTasks, err := GetBaseTasksFromPatchID(r.sc, *at.Version)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting base tasks for task %s: %s", *at.Id, err.Error()))
+	}
+	for _, bT := range baseTasks {
+		if bT.DisplayName == *at.DisplayName {
```

Comment:
this should also check that the variants are the same, because you'll often have the same task name in different variants. Though there is a FindTaskOnBaseCommit function that will do all this for you, which would make this simpler